autoprotocol/protocol.py

```diff
@@ -1677,7 +1677,9 @@ def acoustic_transfer(
             if v > Unit(0, "microliter"):
                 transfers.append(xfer)
             if self.propagate_properties:
-                d.set_properties(s.properties)
+                with warnings.catch_warnings():
+                    warnings.simplefilter("ignore")
+                    d.add_properties(s.properties)
```

Comment:
for now, let's only worry about adding properties. let's remove stuff around warnings, assuming our current implementations can be patched to silence these warnings if we encounter them